### DISCUSSION TITLE
New class in menu item

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#785] [Q&A]
Hi,
I noticed a new class, _.my-1_, has been added to the `<li>` tag in template One (for menu items in the menu module type). I'm wondering if this was a necessary change...

I've encountered [similar unexpected modifications](https://github.com/templaza/astroid-framework/discussions/639) in my templates after updates to template One in the past, it can be a bit frustrating.

Is there a template change log available that I can reference to avoid unexpected changes and save time?

Thanks,
Lorenzo

**Comment by Chacapamac** [comment]
Template Changes list should be available at the Update Download File level to be available before updating.

**Reply by iorbita** [reply]
... I can't see where 🙄

**Comment by Chacapamac** [comment]
Yeah Sorry @iorbita , my last post was a suggestion to include all template changes  of an update. I’m sure Github permit that but me also, I don’t know where to see that?

**Reply by iorbita** [reply]
... I think it's important to have a changelog for templates too, thanks!

**Comment by sonvnn** [comment]
@iorbita Sorry for the late reply. Since 3.1.0 One and Zero will load html from Framework instead of Template. That's why you can see the change on the front-end. 

You can check the changelog here https://github.com/templaza/astroid-framework/commit/d6c4fd1b6b30ada84475adf6a2559ba681a02608#diff-d1aae6d5054965746450242fb78e7e2c2815405d72841e4bc79641e88833456d

Please help me clear Astroid Cache to recompile css. It will stable in the next versions.

Best Regards,
Sonny

**Reply by Chacapamac** [reply]
I knew Github will have a complete changelog. Thanks @sonvnn

———————————————————————————————————————————
**HOW TO CHECK FULL CHANGELOG OF A VERSION**

• Go to the “**Releases**”
![release](https://github.com/user-attachments/assets/6cddd531-5b64-4ead-aee6-457542348912)

• Choose the version and click
![Changelog](https://github.com/user-attachments/assets/aa01dadd-756b-4859-95db-712737d535a3)

**Comment by iorbita** [comment]
I see, but that makes it harder to locate a template update.
In my case, how to know that a class has been added...?

**Reply by Chacapamac** [reply]
You right, I do not see a changelog specific for the templates?

**Reply by sonvnn** [reply]
Since 3.1.0 Menu Module will load from Framework. You can see it here https://github.com/templaza/astroid-framework/blob/master/framework/html/mod_menu/default.php#L29
